src/app/doctor/[id]/doctorPage.module.css

```
@@ -0,0 +1,157 @@
+.container {
+  background-color: var(hsl(200deg 10% 70%));
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  padding: 20px 150px;
+  .left {
```

بین Declaration و Selector باید یه خط خالی فاصله باشه
خیلی جاها رعایت نشده، همه رو اصلاح کنید

```suggestion
  padding: 20px 150px;
  
  .left {
```